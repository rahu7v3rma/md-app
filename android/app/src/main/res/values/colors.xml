<?xml version="1.0" encoding="utf-8"?>
<resources>
    <color name="primary">#72996B</color>
    <color name="splash">#85B085</color>
</resources>
