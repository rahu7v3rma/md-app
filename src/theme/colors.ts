// theme colors
const PRIMARY = '#72996B';
const PRIMARY_LIGHT = '#D3E29F40';
const BACKGROUND = '#F5F5F5';
const SHEET_BACKGROUND_COLOR = 'rgba(0,0,0,0.5)';
const BACKGROUND_LIGHTEST = '#FAFAFF';
const LOG_PAGE_BACKGROUND_COLOR = '#f5f4ee';

// INPUT
const INPUT = '#D3E6F8';
const INPUT_BG = '#EEF4FA';
const INPUT_ERROR_BG = '#FFECEC';
const INPUT_ERROR_BORDER = '#F9D4D4';
const INPUT_ERROR_PLACEHOLDER = '#D05151';
const INPUT_BORDER = '#D3E29F';

// PROGRESS BAR
const PROGRESS_BAR_FILL = '#4DBA0B';
const PROGRESS_BAR_FILL_BORDER = '#3A9A00';
const PROGRESS_BAR_ACTIVITY_INDICATOR = PRIMARY;
const PROGRESS_BAR_FILL_GREEN = '#72996B';
const PROGRESS_BAR_FILL_DISABLE = '#BFC4C9';
const PROGRESS_BAR_TRACK_GREEN = '#EDF0DA';

// PROGRESS CIRCLE
const PROGRESS_CIRCLE_COLOR = '#f00';
const PROGRESS_CIRCLE_SHADOW_COLOR = '#999';
const PROGRESS_CIRCLE_BG_COLOR = '#e9e9ef';
const PROGRESS_CIRCLE_START_BALL_COLOR = '#72996B';
const PROGRESS_CIRCLE_START_BALL_DISABLE_COLOR = '#BDBECA';
const PROGRESS_CIRCLE_INNER_BALL_COLOR = '#5B37BE';
const PROGRESS_CIRCLE_INNER_BALL_DISABLE_COLOR = '#A4AAAF';

// SLIDER
const SLIDER_BG = 'rgba(120, 120, 128, 0.32)';

// BUTTON
const BUTTON_BORDER = '#D3E6F8';
const GREEN_CHECK_DARK = '#D3E6F8';
const BUTTON_LIGHTER_BORDER = '#f4f8e7';
const BUTTON_LIGHTER_BACKGROUND = '#f4f8e7';
const BUTTON_PRIMARY_BORDER = '#5A3DBF';
const BUTTON_DISABLED_BG = '#cccccc';
const BUTTON_DISABLED_BORDER = '#D6D6D6';
const BUTTON_IMAGE_BACKGROUND = '#F0E957';
const BUTTON_ICON_BACKGROUND = '#D3E29F40';
const BUTTON_RED_BACKGROUND = '#B16663';
const BUTTON_SUCCESS_BACKGROUND = '#4DBA0B';
const BUTTON_GREEN_BACKGROUND = '#72996B';
const BUTTON_WHITE_BACKGROUND = '#FFFFFF';
const BUTTON_DARK_RED_BACKGROUND = '#CD4049';

const TEXT_BLACK = '#271A51'; // more of a dark-blue
const TEXT_DARK_BLACK = '#1F1E22';
const TEXT_BLACK_GRAY = '#4D4F4D';
const TEXT_GRAY_BLACK = '#4D4F4D';
const TEXT_WHITE = '#FFFFFF';
const TEXT_GREEN = '#72996B';
const TEXT_CHARCOAL = '#4D4F4D';
const TEXT_DUSKY_OLIVE = '#847F60';
const TEXT_ERROR = '#D05151';
const TEXT_LIGHTER = 'rgba(235, 235, 245, 0.6)';
const TEXT_DISABLED = '#D6D6D6';
const TEXT_GRAY = '#A4AAAF';
const TEXT_GRAY_BASE = '#58595A';
const TEXT_PRIMARY = PRIMARY;
const TEXT_PURPLE = '#6E51D0';
const TEXT_LINK = '#3ED1FF';
const TEXT_DARK_GREEN = '#72996B';
const TEXT_DARKER = '#4D4F4D';
const TEXT_MAIN_DARKER = '#4D4F4D';

//ICONS
const EYE_ICON = '#6E51D0';
const EYE_ICON_GREEN = '#72996B';
const EYE_ICON_ERROR_BORDER = '#D05151';
const GREEN_CHECK_MARK = '#4DBA0B';

// toast messages
const ERROR_BACKGROUND = '#D05151';
const SUCCESS_BACKGROUND = '#4DBA0B';
const INFO_BACKGROUND = '#A4AAAF';

//Modals
const MODAL_BACKDROP = 'rgba(0,0,0,0.5)';
const MODAL_CLOSE_HANDLE = '#F5F8FB';
const MODAL_ITEM_DIVIDER = '#E6E8EC';

//EXTRAS
const SUCCESS_LIGHTEST = '#ECF8E6';
const SUCCESS_LIGHTER = '#DFEED8';
const DARK_CHARCOAL = '#1F1E22';
const WHITE = '#FFFFFF';
const BLACK = '#000000';
const BLACK_LIGHTER = 'rgba(37, 37, 37, 0.6)';
const JOURNEY_BACKGROUND = '#CEE1FE';
const JOURNEY_SECONDARY = '#ADCAF4';
const BLOCK_IN_PROGRESS = PRIMARY;
//const BLOCK_COMPLETE = '#4DBA0B';
const ERROR_LIGHTEST = '#FFECEC';
const ERROR_LIGHTER = '#F9D4D4';
const ERROR_BASE = '#D05151';
const BLACK_SHADOW = '#171717';
const CARD_COMPLETED = '#eff8e7';
const CARD_INPROGRESS = '#f0f4f8';
const PAGE_BG = '#F5F8FB';
const LIGHT_BACK_COLOR = '#D3E29F40';
const BLACK_CARDS = 'rgba(0, 0, 0, 0.3)';
const HEADER_SHADOW = 'rgba(0, 0, 0, 0.03)';
const GRAY_SHADOW = '#0000000D';
const BORDER_COLOR = '#E1E6C2';
const TRANSPARENT_COLOR = 'transparent';
const BADGE_RED = '#F50157';

//SWITCH
const SWITCH_TOGGLED = '#72996B';
const SWITCH_BUTTON = WHITE;

// pagination
const ACTIVE_PAGINATION_DOT = PRIMARY;
const INACTIVE_PAGINATION_DOT = PRIMARY_LIGHT;

export const Colors = {
    theme: {
        primary: PRIMARY,
        primary_light: PRIMARY_LIGHT,
        app_background: BACKGROUND,
        app_sheet_background_color: SHEET_BACKGROUND_COLOR,
        app_background_lightest: BACKGROUND_LIGHTEST,
        log_page_background_color: LOG_PAGE_BACKGROUND_COLOR
    },
    text: {
        black: TEXT_BLACK,
        black_gray: TEXT_BLACK_GRAY,
        dark_green: TEXT_DARK_GREEN,
        text_gray_black: TEXT_GRAY_BLACK,
        dark_black: TEXT_DARK_BLACK,
        white: TEXT_WHITE,
        error: TEXT_ERROR,
        green: TEXT_GREEN,
        charcoal: TEXT_CHARCOAL,
        dusky_olive: TEXT_DUSKY_OLIVE,
        text_lighter: TEXT_LIGHTER,
        text_disabled: TEXT_DISABLED,
        gray: TEXT_GRAY,
        gray_Base: TEXT_GRAY_BASE,
        primary: TEXT_PRIMARY,
        purple: TEXT_PURPLE,
        link: TEXT_LINK,
        darker: TEXT_DARKER,
        mainDarker: TEXT_MAIN_DARKER
    },
    input: {
        error_bg: INPUT_ERROR_BG,
        error_border: INPUT_ERROR_BORDER,
        app_input: INPUT,
        app_input_bg: INPUT_BG,
        input_placeholder_error: INPUT_ERROR_PLACEHOLDER,
        input_border: INPUT_BORDER,
        light_back_color: LIGHT_BACK_COLOR
    },
    button: {
        app_button_border: BUTTON_BORDER,
        app_button_lighter_background: BUTTON_LIGHTER_BACKGROUND,
        app_button_lighter_border: BUTTON_LIGHTER_BORDER,
        app_button_primary_border: BUTTON_PRIMARY_BORDER,
        app_button_disabled_bg: BUTTON_DISABLED_BG,
        app_button_disabled_border: BUTTON_DISABLED_BORDER,
        app_button_background: BUTTON_IMAGE_BACKGROUND,
        app_icon_button_background: BUTTON_ICON_BACKGROUND,
        app_button_red_background: BUTTON_RED_BACKGROUND,
        app_button_success_background: BUTTON_SUCCESS_BACKGROUND,
        app_button_green_background: BUTTON_GREEN_BACKGROUND,
        app_button_white_background: BUTTON_WHITE_BACKGROUND,
        app_btn_light_color: LIGHT_BACK_COLOR,
        app_button_dark_red_background: BUTTON_DARK_RED_BACKGROUND
    },
    icons: {
        eye: EYE_ICON,
        eyeGreen: EYE_ICON_GREEN,
        eye_error_border: EYE_ICON_ERROR_BORDER,
        green_check_mark: GREEN_CHECK_MARK,
        green_dark_check: GREEN_CHECK_DARK
    },
    modals: {
        backdrop: MODAL_BACKDROP,
        close_handle: MODAL_CLOSE_HANDLE,
        divider: MODAL_ITEM_DIVIDER
    },
    toast: {
        errorBackground: ERROR_BACKGROUND,
        successBackground: SUCCESS_BACKGROUND,
        infoBackground: INFO_BACKGROUND
    },
    progress: {
        fillWhite: TEXT_WHITE,
        fill: PROGRESS_BAR_FILL,
        fill_border: PROGRESS_BAR_FILL_BORDER,
        activity_indicator: PROGRESS_BAR_ACTIVITY_INDICATOR,
        fillGreen: PROGRESS_BAR_FILL_GREEN,
        fillDisable: PROGRESS_BAR_FILL_DISABLE,
        trackColor: PROGRESS_BAR_TRACK_GREEN,
        headerProgressColor: PROGRESS_CIRCLE_INNER_BALL_COLOR
    },
    slider: {
        bg: SLIDER_BG
    },
    extras: {
        success_lightest: SUCCESS_LIGHTEST,
        success_lighter: SUCCESS_LIGHTER,
        white: WHITE,
        black: BLACK,
        black_lighter: BLACK_LIGHTER,
        blue_background: JOURNEY_BACKGROUND,
        blue_secondary: JOURNEY_SECONDARY,
        block_in_progress: BLOCK_IN_PROGRESS,
        block_complete: BLOCK_IN_PROGRESS,
        error_lighter: ERROR_LIGHTER,
        error_lightest: ERROR_LIGHTEST,
        error_base: ERROR_BASE,
        black_shadow: BLACK_SHADOW,
        card_completed: CARD_COMPLETED,
        card_inProgress: CARD_INPROGRESS,
        page_bg: PAGE_BG,
        blackCards: BLACK_CARDS,
        headerShadow: HEADER_SHADOW,
        dark_charcoal: DARK_CHARCOAL,
        block_locked: INPUT_BORDER,
        gray_shadow: GRAY_SHADOW,
        border_color: BORDER_COLOR,
        transparent: TRANSPARENT_COLOR,
        badgeRed: BADGE_RED
    },
    pagination: {
        active_dot: ACTIVE_PAGINATION_DOT,
        inactive_dot: INACTIVE_PAGINATION_DOT
    },
    progressCircle: {
        color: PROGRESS_CIRCLE_COLOR,
        shadow_color: PROGRESS_CIRCLE_SHADOW_COLOR,
        bg_color: PROGRESS_CIRCLE_BG_COLOR,
        start_ball_color: PROGRESS_CIRCLE_START_BALL_COLOR,
        start_ball_disable_color: PROGRESS_CIRCLE_START_BALL_DISABLE_COLOR,
        inner_ball_color: PRIMARY,
        inner_ball_disable_color: PROGRESS_CIRCLE_INNER_BALL_DISABLE_COLOR
    },
    switch: {
        button: SWITCH_BUTTON,
        toggled: SWITCH_TOGGLED
    },
    statusBar: {
        white: WHITE
    }
};
