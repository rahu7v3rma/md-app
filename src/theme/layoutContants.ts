export const LayoutContants = {
    gapSpacing: 5,
    cardHeight: 90
};
