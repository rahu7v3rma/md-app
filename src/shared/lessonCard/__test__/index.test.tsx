import React from 'react';
import { StyleSheet } from 'react-native';
import renderer, { act } from 'react-test-renderer';

import LessonCard from '@/shared/lessonCard';
import { Colors } from '@/theme/colors';
import { LayoutContants } from '@/theme/layoutContants';

jest.mock('react-native-modal', () => 'react-native-modal');

const lessonCardStyle = StyleSheet.create({
    card: {
        width: '100%',
        height: LayoutContants.cardHeight,
        borderColor: Colors.button.app_button_disabled_bg,
        borderRadius: 20,
        flexDirection: 'row',
        justifyContent: 'space-between',
        alignItems: 'center',
        marginBottom: 10,
        backgroundColor: Colors.extras.white,
        elevation: 1,
        shadowOpacity: 0.06,
        shadowRadius: 3,
        shadowOffset: { width: 0, height: 0 },
        shadowColor: Colors.theme.app_sheet_background_color
    },
    cardComplete: {},
    cardInProgress: {
        backgroundColor: Colors.extras.card_inProgress
    }
});

it('lessonCard snapshot', async () => {
    const tree = renderer.create(<LessonCard />);
    expect(tree).toMatchSnapshot();
});

it('lessonCard renders without crashing', async () => {
    const tree = renderer.create(<LessonCard />);
    expect(tree.root).toBeTruthy();
});

it('displays progress animation when progress is true', async () => {
    const tree = renderer.create(
        <LessonCard cardContainerID={'cardContainerID'} progress={true} />
    );
    let cardContainerID = tree.root.findByProps({
        testID: 'cardContainerID'
    }).props;
    expect(cardContainerID.style).toStrictEqual([
        lessonCardStyle.card,
        undefined,
        lessonCardStyle.cardInProgress,
        {
            transform: expect.anything()
        },
        undefined
    ]);
});

it('displays completion animation when complete is true', async () => {
    const tree = renderer.create(
        <LessonCard cardContainerID={'cardContainerID'} complete={true} />
    );
    let cardContainerID = tree.root.findByProps({
        testID: 'cardContainerID'
    }).props;
    expect(cardContainerID.style).toStrictEqual([
        lessonCardStyle.card,
        lessonCardStyle.cardComplete,
        undefined,
        {
            transform: expect.anything()
        },
        undefined
    ]);
});

it('calls onPress when the card is pressed', async () => {
    const onPress = jest.fn();
    const tree = renderer.create(
        <LessonCard
            cardTouchViewID={'cardTouchViewID'}
            complete={true}
            onPress={onPress}
        />
    );
    let cardTouchViewID = tree.root.findByProps({
        testID: 'cardTouchViewID'
    }).props;
    await act(() => cardTouchViewID.onPress());
    expect(onPress).toHaveBeenCalledTimes(1);
});

it('calls onCompleted with true when the card is completed', async () => {
    const onCompleted = jest.fn();
    await act(async () => {
        renderer.create(
            <LessonCard
                cardTouchViewID={'cardTouchViewID'}
                complete={true}
                onCompleted={onCompleted}
                shouldAnimate={false}
            />
        );
    });
    expect(onCompleted).toHaveBeenCalledTimes(1);
});
