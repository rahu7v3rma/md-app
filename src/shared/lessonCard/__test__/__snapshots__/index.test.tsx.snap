// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lessonCard snapshot 1`] = `
<View
  collapsable={false}
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#FFFFFF",
      "borderColor": "#cccccc",
      "borderRadius": 20,
      "elevation": 1,
      "flexDirection": "row",
      "height": 90,
      "justifyContent": "space-between",
      "marginBottom": 10,
      "shadowColor": "rgba(0,0,0,0.5)",
      "shadowOffset": {
        "height": 0,
        "width": 0,
      },
      "shadowOpacity": 0.06,
      "shadowRadius": 3,
      "transform": [
        {
          "translateX": 0,
        },
      ],
      "width": "100%",
    }
  }
>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": true,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={false}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      {
        "alignItems": "center",
        "backgroundColor": "#FFFFFF",
        "borderRadius": 20,
        "flexDirection": "row",
        "height": 90,
        "justifyContent": "space-between",
        "opacity": 1,
        "width": "100%",
      }
    }
  >
    <View
      collapsable={false}
      style={
        {
          "backgroundColor": "#FFFFFF",
          "borderRadius": 20,
          "height": 100,
          "left": 0,
          "position": "absolute",
          "top": 0,
          "width": "0%",
        }
      }
    />
    <View
      style={
        {
          "alignItems": "center",
          "flexDirection": "row",
          "width": "70%",
        }
      }
    >
      <View
        style={
          {
            "alignItems": "center",
            "backgroundColor": "#f8f8f8",
            "borderRadius": 15,
            "height": 70,
            "justifyContent": "center",
            "marginLeft": 10,
            "width": "28%",
          }
        }
      >
        <Image
          resizeMode="contain"
          source={
            {
              "uri": undefined,
            }
          }
          style={
            {
              "height": 45,
              "width": 45,
            }
          }
        />
      </View>
      <View
        style={
          {
            "marginStart": 10,
            "width": "65%",
          }
        }
      >
        <Text
          adjustsFontSizeToFit={true}
          style={
            [
              {
                "fontFamily": "Poppins",
              },
              undefined,
              {
                "color": "#4D4F4D",
                "fontSize": 14,
                "fontWeight": "700",
                "lineHeight": 22,
              },
            ]
          }
        />
        <Text
          adjustsFontSizeToFit={true}
          style={
            [
              {
                "fontFamily": "Poppins",
              },
              {
                "marginTop": 5,
              },
              {
                "color": "#A4AAAF",
                "fontSize": 12,
                "fontWeight": undefined,
                "lineHeight": 20,
              },
            ]
          }
        />
      </View>
    </View>
    <View
      style={
        {
          "alignItems": "flex-end",
          "paddingLeft": 10,
          "paddingRight": 10,
          "width": "30%",
        }
      }
    >
      <View>
        <View>
          <RNSVGSvgView
            align="xMidYMid"
            bbHeight={25}
            bbWidth={25}
            fill="none"
            focusable={false}
            height={25}
            meetOrSlice={0}
            minX={0}
            minY={0}
            style={
              [
                {
                  "backgroundColor": "transparent",
                  "borderWidth": 0,
                },
                {
                  "flex": 0,
                  "height": 25,
                  "width": 25,
                },
              ]
            }
            vbHeight={24}
            vbWidth={24}
            width={25}
            xml="<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
<path d="M12 17.35C12.9002 17.35 13.63 16.6202 13.63 15.72C13.63 14.8198 12.9002 14.09 12 14.09C11.0998 14.09 10.37 14.8198 10.37 15.72C10.37 16.6202 11.0998 17.35 12 17.35Z" fill="#A4AAAF"/>
<path d="M18.28 9.53V8.28C18.28 5.58 17.63 2 12 2C6.37 2 5.72 5.58 5.72 8.28V9.53C2.92 9.88 2 11.3 2 14.79V16.65C2 20.75 3.25 22 7.35 22H16.65C20.75 22 22 20.75 22 16.65V14.79C22 11.3 21.08 9.88 18.28 9.53ZM12 18.74C10.33 18.74 8.98 17.38 8.98 15.72C8.98 14.05 10.34 12.7 12 12.7C13.66 12.7 15.02 14.06 15.02 15.72C15.02 17.39 13.67 18.74 12 18.74ZM7.35 9.44C7.27 9.44 7.2 9.44 7.12 9.44V8.28C7.12 5.35 7.95 3.4 12 3.4C16.05 3.4 16.88 5.35 16.88 8.28V9.45C16.8 9.45 16.73 9.45 16.65 9.45H7.35V9.44Z" fill="#A4AAAF"/>
</svg>
"
            xmlns="http://www.w3.org/2000/svg"
          >
            <RNSVGGroup
              fill={null}
              propList={
                [
                  "fill",
                ]
              }
            >
              <RNSVGPath
                d="M12 17.35C12.9002 17.35 13.63 16.6202 13.63 15.72C13.63 14.8198 12.9002 14.09 12 14.09C11.0998 14.09 10.37 14.8198 10.37 15.72C10.37 16.6202 11.0998 17.35 12 17.35Z"
                fill={4288981679}
                propList={
                  [
                    "fill",
                  ]
                }
              />
              <RNSVGPath
                d="M18.28 9.53V8.28C18.28 5.58 17.63 2 12 2C6.37 2 5.72 5.58 5.72 8.28V9.53C2.92 9.88 2 11.3 2 14.79V16.65C2 20.75 3.25 22 7.35 22H16.65C20.75 22 22 20.75 22 16.65V14.79C22 11.3 21.08 9.88 18.28 9.53ZM12 18.74C10.33 18.74 8.98 17.38 8.98 15.72C8.98 14.05 10.34 12.7 12 12.7C13.66 12.7 15.02 14.06 15.02 15.72C15.02 17.39 13.67 18.74 12 18.74ZM7.35 9.44C7.27 9.44 7.2 9.44 7.12 9.44V8.28C7.12 5.35 7.95 3.4 12 3.4C16.05 3.4 16.88 5.35 16.88 8.28V9.45C16.8 9.45 16.73 9.45 16.65 9.45H7.35V9.44Z"
                fill={4288981679}
                propList={
                  [
                    "fill",
                  ]
                }
              />
            </RNSVGGroup>
          </RNSVGSvgView>
          <View
            collapsable={false}
            style={
              {
                "opacity": 0,
                "position": "absolute",
              }
            }
          >
            <RNSVGSvgView
              align="xMidYMid"
              bbHeight={25}
              bbWidth={25}
              fill="none"
              focusable={false}
              height={25}
              meetOrSlice={0}
              minX={0}
              minY={0}
              style={
                [
                  {
                    "backgroundColor": "transparent",
                    "borderWidth": 0,
                  },
                  {
                    "flex": 0,
                    "height": 25,
                    "width": 25,
                  },
                ]
              }
              vbHeight={24}
              vbWidth={24}
              width={25}
              xml="<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
<path d="M12 17.35C12.9002 17.35 13.63 16.6202 13.63 15.72C13.63 14.8198 12.9002 14.09 12 14.09C11.0998 14.09 10.37 14.8198 10.37 15.72C10.37 16.6202 11.0998 17.35 12 17.35Z" fill="#A4AAAF"/>
<path d="M18.28 9.53V8.28C18.28 5.58 17.63 2 12 2C6.37 2 5.72 5.58 5.72 8.28V9.53C2.92 9.88 2 11.3 2 14.79V16.65C2 20.75 3.25 22 7.35 22H16.65C20.75 22 22 20.75 22 16.65V14.79C22 11.3 21.08 9.88 18.28 9.53ZM12 18.74C10.33 18.74 8.98 17.38 8.98 15.72C8.98 14.05 10.34 12.7 12 12.7C13.66 12.7 15.02 14.06 15.02 15.72C15.02 17.39 13.67 18.74 12 18.74ZM7.35 9.44C7.27 9.44 7.2 9.44 7.12 9.44V8.28C7.12 5.35 7.95 3.4 12 3.4C16.05 3.4 16.88 5.35 16.88 8.28V9.45C16.8 9.45 16.73 9.45 16.65 9.45H7.35V9.44Z" fill="#A4AAAF"/>
</svg>
"
              xmlns="http://www.w3.org/2000/svg"
            >
              <RNSVGGroup
                fill={null}
                propList={
                  [
                    "fill",
                  ]
                }
              >
                <RNSVGPath
                  d="M12 17.35C12.9002 17.35 13.63 16.6202 13.63 15.72C13.63 14.8198 12.9002 14.09 12 14.09C11.0998 14.09 10.37 14.8198 10.37 15.72C10.37 16.6202 11.0998 17.35 12 17.35Z"
                  fill={4288981679}
                  propList={
                    [
                      "fill",
                    ]
                  }
                />
                <RNSVGPath
                  d="M18.28 9.53V8.28C18.28 5.58 17.63 2 12 2C6.37 2 5.72 5.58 5.72 8.28V9.53C2.92 9.88 2 11.3 2 14.79V16.65C2 20.75 3.25 22 7.35 22H16.65C20.75 22 22 20.75 22 16.65V14.79C22 11.3 21.08 9.88 18.28 9.53ZM12 18.74C10.33 18.74 8.98 17.38 8.98 15.72C8.98 14.05 10.34 12.7 12 12.7C13.66 12.7 15.02 14.06 15.02 15.72C15.02 17.39 13.67 18.74 12 18.74ZM7.35 9.44C7.27 9.44 7.2 9.44 7.12 9.44V8.28C7.12 5.35 7.95 3.4 12 3.4C16.05 3.4 16.88 5.35 16.88 8.28V9.45C16.8 9.45 16.73 9.45 16.65 9.45H7.35V9.44Z"
                  fill={4288981679}
                  propList={
                    [
                      "fill",
                    ]
                  }
                />
              </RNSVGGroup>
            </RNSVGSvgView>
          </View>
        </View>
      </View>
    </View>
  </View>
</View>
`;
